scripts/ghidra/decompile-headless.sh

```diff
@@ -19,11 +19,14 @@ Options:
   -v, --verbose         Enable verbose output
   -t, --interactive     Start Docker container in interactive mode
   -c, --ci              Run in CI mode
+  --test                Run tests in the Docker container
```

Comment:
will NOT be removed but will use the test container instead of the headless container directly